src/app/components/JumpTo/index.tsx

```diff
@@ -0,0 +1,58 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import Text from '#app/components/Text';
+import InlineLink from '#app/components/InlineLink';
+import idSanitiser from '../../lib/utilities/idSanitiser';
+
+export interface JumpToProps {
+  jumpToHeadings?: Array<{ heading: string }>;
+  eventTrackingData?: EventTrackingMetadata;
+}
+
+const JumpTo = ({ jumpToHeadings, eventTrackingData }: JumpToProps) => {
+  const { translations } = useContext(ServiceContext);
+  const { jumpTo = 'Jump to' } = translations?.articlePage || {};
+
+  const viewRef = useViewTracker(eventTrackingData);
+  const clickTrackerHandler = useClickTrackerHandler({
+    ...eventTrackingData,
+    componentName: 'jumpto',
+  });
+
+  const headingId = 'jump-to-heading';
+
+  return (
+    <nav
+      ref={viewRef}
+      role="navigation"
+      aria-labelledby={headingId}
```

Comment:
Can we update {headingId} to {strongId} please

Reply:
Yes! Renaming it to {titleId} so it is semantically correct